pennylane/qchem/structure.py

```diff
@@ -412,3 +413,130 @@ def excitations_to_wires(singles, doubles, wires=None):
         doubles_wires.append([d1_wires, d2_wires])
 
     return singles_wires, doubles_wires
+
+
+def mol_data_from_pubchem(identifier, identifier_type="name", cid=None):
+    r"""Obtain molecular data of the compound from the PubChem Database
+
+    Args:
+        identifier (str): compound's idenitifer as required by the PubChem database
+        identifier_type (str): type of chemical indentifier - name, CAS, SMILES, InChI, InChIKey
+        cid (int): PubChem CID for the molecule. `identifier` will be ignored if this is not ``None``.
+
+    Returns:
+        Tuple(list(str), ndarray, int): symbols, geometry and charge of the compound
+
+    **Example**
+
+    >>> mol_data_from_pubchem("BeH2")
+    (['Be', 'H', 'H'],
+     array([[ 4.79405604,  0.29290815,  0.        ],
+            [ 3.77946   , -0.29290815,  0.        ],
+            [ 5.80884105, -0.29290815,  0.        ]]),
+     0)
+    >>> mol_data_from_pubchem("NH4", cid=223)
```

Comment:
It seems a bit strange to me that
`qml.qchem.mol_data_from_pubchem("H2O", cid=223)`
still returns Ammonia. 

Is it possible to make the `cid` an identifier type? So, the identifier could be either `name`, `CAS`, ..., and `CID`.